src/drive/web/modules/navigation/RealtimeFiles.jsx

```diff
@@ -18,23 +19,14 @@ import logger from 'lib/logger'
 export class RealtimeFiles extends React.Component {
   realtimeListener = null
   pouchListener = null
-  static contextTypes = {
-    client: PropTypes.object.isRequired
-  }
+
   componentWillMount() {
```

Comment:
Could we change this to `didMount` while we're here, or what that involve other changes?